src/metabase/pulse/render/table.clj

```diff
@@ -74,12 +74,18 @@
   [score]
   (int (* (/ score 100.0) max-bar-width)))
 
+(defn- truncate-text [text]
```

Comment:
I tried searching through out the project, but couldn't find if we have such utility. I found an example where we just create a private function just for this.
https://github.com/metabase/metabase/blob/eb30d164488c63a56dbce77404491e05ee90f4c0/enterprise/backend/src/metabase_enterprise/serialization/v2/utils/yaml.clj#L8-L11

Reply:
Thanks for the suggestion 👍 I did what you suggested, and it looks a lot simpler.